apps/microtvm-vm/base-box/setup-tvm-user.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Zephyr
+pip3 install --user -U west
+echo 'export PATH=$HOME/.local/bin:"$PATH"' >> ~/.profile
+source ~/.profile
+echo PATH=$PATH
+west init --mr v2.4.0 ~/zephyr
+cd ~/zephyr
+west update
+west zephyr-export
+
```

Comment:
This appears to be zephyr specific and not uTVM specific. Is there really an expectation that the vagrant box will have a user named west ? 

What happens when we need other RTOS's ?

Reply:
actually `west` is the Zephyr command-line tool, not a username. 

adding other RTOSes may present a challenge. i'd advocate we cross that bridge when we get there--likely, if we need to maintain multiple python venv's, it would be something like we pip install west into a venv, then symlink west to the venv's bin. I don't know if that will fully work, but we'll have to see.

Reply:
i've thought about this some more and I'd like to actually consider this VM to be Zephyr-specific. I'd rather allow platforms the freedom to integrate in the way that makes sense for them. I'll update the path to this VM to be `apps/microtvm-vm/zephyr/base-box`, and provide a `README.md` explaining about the subdirectories in this tree.

Reply:
Apologies, it also looks like I might have misparsed the `pip3 install --user -U west' into thinking that it was specific for a `west' user , re-reading it again suggests my expectation there was a bit misplaced. 




regards
Ramana